scripts/eurostat/gdp/us_state_gdp_quarterly_import_config.txt

```diff
@@ -0,0 +1,14 @@
+import: {
+  import_name: "eurostatGDPImport3"
+  provenance_url: "https://ec.europa.eu/eurostat/data/database"
+  provenance_description: "Yearly data on GDP in Europe from Eurostat"
+  # Path to the resolved MCF. Can be a repeated field.
+  mcf_url: "gs://resolved_mcf/external_tables/eurostatGDPImport7/nodes.mcf-00000-of-00001"
+  # First date of imported data.
+  start_date: "2000"
+  # Last date of imported data.
+  end_date: "2018"
+  # Source data release schedule.
+  update_frequency: "P1Y"
+  next_release_date: "Unknown"
```

Comment:
Leave out if unknown. Feel free to do this in a follow up. File a bug in that case.

Reply:
Change made in new branch to be made into a PR soon.